src/packages/graphql.ts

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
this file is same as the one in versions folder. We can reuse the code here.

Reply:
Yes, we can. I duplicated this file since I could not find how are common features organized in other Github actions projects.

So, to help to solve this little issue, I ask... How do you organize common features? Create a `common` folder?

Reply:
Yes we can move it inside a new `common` folder.